### DISCUSSION TITLE
How should one access the RaspberryMatic instance which is running as a HomeAssistant Addon?

### DISCUSSION THREAD

**Discussion by ericleigh007** [OpenCCU/OpenCCU#1735] [🇩🇪 Fragen&Antworten]
My system is a Home Assistant OS 7.4 system using a RaspberryMatic as an add-on?

How, if it is possible, can i access the RaspberryMatic instance in order to use it in my Smartha app?

The configuration is:

1. Windows Host
2. VMware Workstation Player 16.1
3. HomeAssistant OS 7.4
4. RasperryMatic as an add-on (so in a docker container)

In Smartha, it asks for an IP address, which I don't know how to determine, as well as a username and password (which I believe i have)

This may help others who are trying to use RaspberryMatic (which it seems should work this way)

-thanks
-e

**Comment by Baxxy13** [comment]
The IP-Address is the one from your Homeassistant VM. 
But to connect to the RaspberryMatic AddOn you must first "route out" the needed ports via the RM-AddOn Configuration Options.
I don't know which ports are needed by Smartha-App. You should ask for that in Smartha Forum.

**Reply by ericleigh007** [reply]
How would one "route out" the ports?  I have seen that configuration and it doesn't seem obvious what to do, including in the help files.

If I know what ports I need to route out, what do I place in the blanks next to the ports?  The display comes up "Disabled", but not sure how to enable.

I think in this case, the help could use some work, since I'm probably not the only newbie who doesn't understand that config page.

-thanks
-e

**Comment by Columbus1964** [comment]
Hello, I opened a similar discussion a day ago. 
(RaspberryMatic als HA Addon - Zugriff TinyMatic oder anderer Drittanbieter-Apps #1774)

Unfortunately, I only discovered this one now and I have no solution.
I hope, someone can help us.
the only way I see, run Raspberrymatic not as Addon, but as a second VM with own IP

Best regards
Columbus1964

**Reply by ericleigh007** [reply]
Yes, again, route out by doing what? 

say for instance i need to route out port 2010.  How do I set that up in the configuration page (as far as I've seen, there is no help entry for that page and I'll gladly create it once i understand its functioning) .

Let's close this loop and help me, @Columbus1964, and everyone else who struggles here, THANKS!

**Reply by Columbus1964** [reply]
Hello ericleigh007,

do you speak a little bit german? If yes, please read the answer in my discussion "RaspberryMatic als HA Addon - Zugriff TinyMatic oder anderer Drittanbieter-Apps https://github.com/jens-maus/RaspberryMatic/discussions/1774"
There is the solution for our problem..
Maybe I can translate to English for you, but my English is not so good,
Best regards
Columbus1964
